src/components/form-tags/_form-tags.scss

```diff
@@ -1,4 +1,12 @@
 .b-form-tags {
+  > ul {
+    margin-top: -0.25rem;
+
+    > li {
+      margin-top: 0.25rem;
+    }
+  }
+
```

Comment:
Could this conflict with custom rendering? 
Would it be better to add a class to the default rendered `ul` and target that instead?

Reply:
@Hiws Yeah it could. Maybe we should add ad `b-form-tags-list` class to the list. We should also replace `> li {}` with `.b-form-tag {}`.